modules/preparing-an-inital-cluster-deployment-for-mce-connected.adoc

```diff
@@ -0,0 +1,297 @@
+// Module included in the following assemblies:
+//
+// * installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-an-inital-cluster-deployment-for-mce-connected_{context}"]
+
+= Preparing an initial cluster deployment for multicluster engine for Kubernetes(MCE) Operator while connected
```

Comment:
```suggestion
= Preparing to install multicluster engine for Kubernetes (MCE) Operator while connected
```
Is there a way you can shorten this heading? Just a suggestion, but it seems a bit long to me.
Also I think there should be a space before the acronym